zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -157,10 +157,11 @@ public void testRemoveButton() throws Exception {
       sleep(1000, true);
       driver.findElement(By.xpath("//div[@class='modal-dialog'][contains(.,'delete this paragraph')]" +
           "//div[@class='modal-footer']//button[contains(.,'OK')]")).click();
+      sleep(1000, true);
```

Comment:
Is there a reason to use this method vs `ZeppelinITUtils.sleep(1000, false);` below in the same method?

If they are the same, may be while we are here it can be refactored to use the same impl?